### DISCUSSION TITLE
Setting up Bakery

### DISCUSSION THREAD

**Discussion by TechplexEngineer** [gokrazy/gokrazy#228] [Q&A] [answered]
Hallo,

I'm working on boot performance testing with changing switches in the kernel config. I'd like to setup a version of the bakery to test out builds.

I swear I read somewhere that described the setup and wiring but can't seem to locate it.

Does anyone have a link to the blog post or article describing it?

https://github.com/gokrazy/bakery

Thanks!

**Comment by stapelberg** [comment, marked as answer]
Hey! I don’t think there’s a blog post or written-down explanation anywhere. I remember that I mentioned the bakery setup in talks or streams, though.

Is there a particular step you’re having trouble with? Setup should be relatively straight-forward. If in doubt, you can consult the source, which is relatively brief.

**Reply by TechplexEngineer** [reply]
Ah, that's why I couldn't find it.

I was hoping to find what you were using to turn on and off the pi's. Looks like a tasmota device. https://github.com/gokrazy/bakery/blob/main/cmd/bootery/bootery.go#L725

**Reply by stapelberg** [reply]
Yep, it’s a Sonoff S26 running Tasmota. See also https://michael.stapelberg.ch/posts/2022-03-19-smart-home-components/#smart-power-plug :)
